app/views/action_plans/show.html.erb

```diff
@@ -3,8 +3,14 @@
 
   <div id="action-plan">
     <h2>Action Plan</h2>
+
+    <label for="action-plan_prefix">
+      Enter Prefix
+      <input name="action-plan_prefix" type="text">
```

Comment:
I think the label's `for` should match the input's `id`
```suggestion
      <input id="action-plan_prefix" name="action-plan_prefix" type="text">
```

Reply:
I wonder if you would also prefer using `-` instead of `_` for the id?

Reply:
@buys-fran I agree with that. I will update the name.